libs/sm/LUNperVDI.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python3
```

Comment:
This should loose the shebang as it's not executable. Basically, general comment, nothing in `libs/sm` should have a shebang

Reply:
Moved them with "git mv". I'll add a commit to the series.

Reply:
> Moved them with "git mv". I'll add a commit to the series.

Annoying that Github doesn't show them like that then :(

Reply:
That's because of the breadcrumb still left in for backwards compatibility. It confuses git a bit unfortunately, but I wanted to keep each commit bisectable in case of issues.